airflow/utils/db.py

```diff
@@ -23,16 +23,18 @@
 from airflow import settings
 from airflow.configuration import conf
 # noinspection PyUnresolvedReferences
-from airflow.jobs.base_job import BaseJob  # noqa: F401  # pylint: disable=unused-import
-# noinspection PyUnresolvedReferences
 from airflow.models import (  # noqa: F401 # pylint: disable=unused-import
     DAG, XCOM_RETURN_KEY, BaseOperator, BaseOperatorLink, Connection, DagBag, DagModel, DagPickle, DagRun,
     DagTag, Log, Pool, SkipMixin, SlaMiss, TaskFail, TaskInstance, TaskReschedule, Variable, XCom,
 )
 # We need to add this model manually to get reset working well
 # noinspection PyUnresolvedReferences
 from airflow.models.serialized_dag import SerializedDagModel  # noqa: F401  # pylint: disable=unused-import
-from airflow.utils.session import create_session, provide_session  # noqa  # pylint: disable=unused-import
+# noinspection PyUnresolvedReferences
+from airflow.models.sqla_models import (  # noqa: F401 # pylint: disable=unused-import
+    BackfillJob, BaseJob, LocalTaskJob, SchedulerJob,
```

Comment:
Is this just so that the migrations etc can find the class? If so we only need BaseJob, not the other three.

Reply:
(I think. Since they are all stored in the "jobs" table the other classes don't affect the table structure in any way)

Reply:
Yes that's for migrations